include/mbgl/renderer/renderer.hpp

```diff
@@ -49,20 +49,14 @@ class Renderer {
                                                  const std::string& extensionField,
                                                  const optional<std::map<std::string, Value>>& args = {}) const;
 
-    void setFeatureState(const std::string& sourceID,
-                         const optional<std::string>& sourceLayerID,
-                         const std::string& featureID,
-                         const FeatureState& state);
-
-    void getFeatureState(FeatureState& state,
-                         const std::string& sourceID,
-                         const optional<std::string>& sourceLayerID,
+    void setFeatureState(const std::string& sourceID, const optional<std::string>& sourceLayerID,
```

Comment:
Personally, I prefer old style (chromium) where each argument is on a new line. @tmpsantos could we add exception for this to clang-format config?

Reply:
@tmpsantos could we add these to clang format?
BinPackParameters: false
AllowAllArgumentsOnNextLine: false